packages/eslint-plugin/docs/rules/no-base-to-string.mdx

```diff
@@ -9,11 +9,11 @@ import TabItem from '@theme/TabItem';
 >
 > See **https://typescript-eslint.io/rules/no-base-to-string** for documentation.
 
-JavaScript will call `toString()` on an object when it is converted to a string, such as when `+` adding to a string or in `${}` template literals.
-The default Object `.toString()` uses the format `"[object Object]"`, which is often not what was intended.
-This rule reports on stringified values that aren't primitives and don't define a more useful `.toString()` method.
+JavaScript will call `toString()` or `toLocaleString()` on an object when it is converted to a string, such as when `+` adding to a string or in `${}` template literals.
```

Comment:
I think JavaScript never implicitly calls `toLocaleString()`, right?

```suggestion
JavaScript will call `toString()` on an object when it is converted to a string, such as when `+` adding to a string or in `${}` template literals.
```

Reply:
You're absolutely right🙇‍♂️ I fixed it in 3ffe462.